source/streaming-mode/streaming-read-config.txt

```diff
@@ -267,3 +281,18 @@ If you create a collection while streaming from all collections, the new
 collection is automatically included in the stream. 
 
 You can drop collections at any time while streaming from multiple collections.
+
+.. important:: Inferring the Schema of a Change Stream
+
+   If you set the ``change.stream.publish.full.document.only``
+   option to ``true``, the {+connector-short+} infers the schema of a ``DataFrame``
+   by using the schema of the scanned documents. If you set the option to
+   ``false``, you must specify a schema.
```

Comment:
[optional]

This part of the note duplicates the note in `streaming-read.txt`. While I find it weird to have such duplication, it's up to the docs team. However, the source code for both notes is also duplicated, which opens opportunities to update one of the notes and not the other, thus causing confusion.

Reply:
There were a few minor differences in the two different admonitions, so unfortunately I couldn't single-source them. If this does begin causing issues we can look at rewording/moving things (or maybe just removing one of the duplicates)